pkg/monitoring/rules/recordingrules/vmi.go

```diff
@@ -41,4 +41,44 @@ var vmiRecordingRules = []operatorrules.RecordingRule{
 		MetricType: operatormetrics.GaugeType,
 		Expr:       intstr.FromString("kubevirt_vmi_memory_available_bytes-kubevirt_vmi_memory_usable_bytes"),
 	},
+	{
+		MetricsOpts: operatormetrics.MetricOpts{
+			Name: "kubevirt_vmi_memory_used_bytes",
+			Help: "Amount of `used` memory as seen by the domain.",
+		},
+		MetricType: operatormetrics.GaugeType,
+		Expr:       intstr.FromString("kubevirt_vmi_memory_available_bytes-kubevirt_vmi_memory_usable_bytes"),
+	},
+	{
+		MetricsOpts: operatormetrics.MetricOpts{
+			Name: "kubevirt_vmi_vcpu_count",
+			Help: "The number of the VMI vCPUs.",
+		},
+		MetricType: operatormetrics.GaugeType,
+		Expr:       intstr.FromString("count by (namespace, name, node) (kubevirt_vmi_vcpu_seconds_total)"),
+	},
+	{
+		MetricsOpts: operatormetrics.MetricOpts{
+			Name: "kubevirt_vmi_guest_vcpu_queue",
```

Comment:
What does this metric actually show? How much more load than vCPUs a VMI had in the last minute?

Reply:
The CPU load metrics include the number of VM vCPUs. This metric subtracts the number of the VCPUs to get the actual CPU queue.

Reply:
So queue is how much more load than vCPUs we have?

Reply:
Yes.

Reply:
The Load metric collected from the guest is a 1m average. It means that this issue persists for at least 1m.
The metric is collected only once a minute, so it could also be for 2m.